backend/django_backend/bookings/tests.py

```diff
@@ -56,3 +59,33 @@ def test_get_available_timeslots(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertIn('timeslots', response.data)
         self.assertIn('interval', response.data)
+
+
+
+class EmailSendTestCase(TestCase):
+    def setUp(self):
+        # Create a user and a service provider
+        self.user = CustomUser.objects.create_user('user@example.com', 'password123', role=CustomUser.USER)
+        self.service_provider = CustomUser.objects.create_user('provider@example.com', 'password123', role=CustomUser.SERVICE_PROVIDER)
+        self.service = Service.objects.create(name="Test Service", service_provider=self.service_provider)
+        
+        # Create a booking
+        self.booking = Booking.objects.create(
+            user=self.user,
+            service_provider=self.service_provider,
+            service=self.service,
+            booking_date="2024-05-10",
+            start_time="10:00",
+            end_time="12:00"
+        )
+        
+        self.client = APIClient()
+        self.client.force_authenticate(user=self.user)
+
+    def test_cancel_booking_sends_email(self):
+        response = self.client.delete(reverse('booking-delete', kwargs={'booking_id': self.booking.id}))
```

Comment:
This won't work after it was mapped to a POST request

Reply:
I think, not sure

Reply:
Could you reproduce test results, send screen pls

Reply:
Tenker uansett vi kan approve denne, siden epost funksjonaliteten bare ligger som et konsept som kan utvides og ikke faktisk fungerende akk nå